backend/app/transcendence/src/socket/socket.modul.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from "@nestjs/common";
+import { SocketEvents } from "./SocketEvents";
+
+@Module(
+{
+	providers: [ SocketEvents ]
+})
+export class SocketModule
```

Comment:
add new line before export maybe?

Reply:
This is a decorator, it is part of the statement  @Module({ providers: [ SocketEvents ]}) export class SocketModule